lib/model/model.go

```
@@ -2096,21 +2096,30 @@ func stringSliceWithout(ss []string, s string) []string {
 // list, we don't also need foo/bar/ because that's already covered.
 func unifySubs(dirs []string, exists func(dir string) bool) []string {
 	var subs []string
+	sep := string(filepath.Separator)
 
 	// Trim each item to itself or its closest known parent
 	for _, sub := range dirs {
-		for sub != "" && sub != ".stfolder" && sub != ".stignore" {
-			if exists(sub) {
-				break
-			}
-			sub = filepath.Dir(sub)
-			if sub == "." || sub == string(filepath.Separator) {
-				// Shortcut. We are going to scan the full folder, so we can
-				// just return an empty list of subs at this point.
-				return nil
+		sub = strings.Trim(sub,sep)
```

Gofmt should fail, missing space after comma